crates/sui-test-validator/src/main.rs

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+
+use clap::{Parser, ValueHint};
+
+use sui_config::genesis_config::GenesisConfig;
+use sui_test_data::create_test_data;
+use test_utils::network::{start_rpc_test_network_with_fullnode, TestNetwork};
+
+/// Start a Sui validator and fullnode for easy testing.
+#[derive(Parser, Debug)]
+#[clap(author, version, about, long_about = None)]
+struct Args {
+    /// Config directory that will be used to store network configuration
+    #[clap(short, long, parse(from_os_str), value_hint = ValueHint::DirPath)]
+    config: Option<std::path::PathBuf>,
+
+    /// If enabled, a set of test transactions will be executed to seed activity on the validator
+    #[clap(long)]
+    with_preset_data: bool,
+
+    /// Port to start the RPC server on
+    #[clap(long, default_value = "9000")]
+    port: u16,
```

Comment:
```suggestion
    #[clap(long, default_value = "5001")]
    gateway_rpc_port: u16,
```

Currently there are two different types of RPC server: Gateway RPC and Fullnode RPC, IIUC, this currently controls the gateway RPC, which has a default port value of 5001.

We should also add another Args for `fullnode_rpc_port`, @patrickkuo / @longbowlu /  @mystenmark do you know how do we configure it with https://github.com/MystenLabs/sui/blob/af2c5ee08b9e7c1b91c1e57c39f0ea21279bfd5d/crates/test-utils/src/network.rs#L39 ?